### PR TITLE
Register Page Front OK

### PR DESCRIPTION
**Pull request by LauraCourquin**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -2,17 +2,22 @@ import Navbar from "@components/Navbar";
...
-      <Route path="/profile/:profileId" element={<Profile />} />
-       </Routes>
-     </div>
 
+      <Navbar />
+      <Routes>
+        <Route path="/profile" element={<Profile />} />
+        <Route path="/register" element={<Register />} />
+        <Route path="/register/artist" element={<RegisterArtist />} /> 
+          <Route path="/profile/:profileId" element={<Profile />} />
```
Comment (LuigiMenez) [inline frontend/src/App.jsx]: Indentation

File: frontend/src/components/DemoProfile/index.jsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (LuigiMenez) [inline frontend/src/components/DemoProfile/index.jsx]: Why?

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Fix minor

**Review by LauraCourquin — COMMENTED** [review]
Update App.jsx

**Review by ghost — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

File: frontend/src/App.jsx [hunk 3]
```diff
@@ -2,17 +2,22 @@ import Navbar from "@components/Navbar";
...
-       </Routes>
-     </div>
 
+      <Navbar />
+      <Routes>
+        <Route path="/profile" element={<Profile />} />
+        <Route path="/register" element={<Register />} />
+        <Route path="/register/artist" element={<RegisterArtist />} /> 
+        <Route path="/profile/:profileId" element={<Profile />} />
+        <Route path="/" element={<Home />} />
```
Comment (LoicBrassart) [inline frontend/src/App.jsx]: I already saw this code one PR ago (and two PR ago, too)
Stop duplicating code (conflict-prone result) and please create your branches *from `dev`*

File: frontend/src/components/Checkbox/index.jsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+      >
+        {act ? (
+          <img src={Black} alt="Bouton check" />
+        ) : (
+          <img src={White} alt="Bouton à checker" />
+        )}
+      </button>
+    </div>
+  );
+}
```
Comment (LoicBrassart) [inline frontend/src/components/Checkbox/index.jsx:21]: ... What's this ?
You created a full component to replace a `<input type="checkbox" />` with an image-in-a-button-in-a-div ? WHY ?! (v°_°v)

File: frontend/src/components/RegisterArtist/index.jsx [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+  const handleChange = () => {
+    setCheckOne(!checkOne);
+    setCheckTwo(false);
+  };
+  const handleChangeBis = () => {
+    setCheckTwo(!checkTwo);
+    setCheckOne(false);
+  };
+
+  const hSubmit = (evt) => {
```
Comment (LoicBrassart) [inline frontend/src/components/RegisterArtist/index.jsx]: Be consistent in your functions' names

File: frontend/src/components/RegisterArtist/index.jsx [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+            <input
+              className="inputForm"
+              type="text"
+              name="secuNum"
+              placeholder="Numéro de sécurité sociale"
+              value={formData.secuNum}
+              onChange={hChangeFormData}
+            />
+            <Link to="/inscription/artiste/artisteSuite">
+              <button className="btnNext" type="submit">
```
Comment (LoicBrassart) [inline frontend/src/components/RegisterArtist/index.jsx]: Isn't it `<input type="submit" />` ?
(I may be wrong on this one, please check it out)

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Globally, beware of code quality and PR reviewing, everybody loses time when one is refused

File: frontend/src/components/Checkbox/index.jsx (hunk 4 above)
Comment (LauraCourquin) [reply frontend/src/components/Checkbox/index.jsx:21]: I was told that checkboxes to style were complicated. 
I went for the simplest.

File: frontend/src/components/RegisterArtist/index.jsx (hunk 6 above)
Comment (LauraCourquin) [reply frontend/src/components/RegisterArtist/index.jsx]: You are right, I will change that.

**Review by LauraCourquin — COMMENTED** [review]
Update index.jsx

**Review by ghost — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

File: frontend/src/components/Checkbox/index.jsx (hunk 4 above)
Comment (LoicBrassart) [reply frontend/src/components/Checkbox/index.jsx:21]: That's not really a good idea: it **looks** better but they're not checkboxes anymore (SEO & accessibility browsers will have a blast trying to explain this !)

**Review by LoicBrassart — DISMISSED** [review]
NmE

**Comment by ghost** [comment]
Resolve conflict with dev App.jsx

File: frontend/src/App.jsx (hunk 3 above)
Comment (ghost) [reply frontend/src/App.jsx]: Done

**Review by ghost — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me

**Review by LoicBrassart — DISMISSED** [review]
NmE

**Review by ghost — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
Much better!

**Pull request closed without merging**